app.js

```diff
@@ -0,0 +1,59 @@
+// import functions and grab DOM elements
+import { userWins, computerMove } from "./combatUtils.js";
+
```

Comment:
Make sure you pay attention to these linting errors and correct them before submitting your lab. They don't always affect how the code runs, but they will prevent your tests from passing.